lib/libutils/isoc/bget_malloc.c

```diff
@@ -262,6 +262,7 @@ static bool bpool_foreach_pool(struct bpool_iterator *iterator, void **buf,
 		*isfree = true;
 
 		/* Assert that the free list links are intact */
+		/* FIXME: assert() or TEE_ASSERT() ? */
```

Comment:
bget natively use `assert()`. Keep using it for bget sanities or move to `TEE_ASSERT()` ?